CHANGELOG.md

```diff
@@ -0,0 +1,447 @@
+# Change Log
+
+## [1.2.10](https://github.com/rime/librime/tree/1.2.10) (2018-02-20)
```

Comment:
The tag referenced here seems not generated yet. Is it necessary to generate ChangeLog after creating the tag?